app/lib/request.rb

```diff
@@ -281,7 +281,7 @@ def private_address_exceptions
 
   class ProxySocket < Socket
     class << self
-      def check_private_address(_address)
+      def check_private_address(_address,_host)
```

Comment:
Just a tiny style issue:
```suggestion
      def check_private_address(_address, _host)
```